matRad_calcPhotonDoseVmc.m

```diff
@@ -0,0 +1,256 @@
+function dij = matRad_calcPhotonDoseVmc(ct,stf,pln,cst,nCasePerBixel,numOfParallelMCSimulations)
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% matRad vmc++ photon dose calculation wrapper
+% 
+% call
+%   dij = matRad_calcPhotonDoseVmc(ct,stf,pln,cst,nCasePerBixel,numOfParallelMCSimulations)
+%
+% input
+%   ct:                         matRad ct struct
+%   stf:                        matRad steering information struct
+%   pln:                        matRad plan meta information struct
+%   cst:                        matRad cst struct
+%   nCasePerBixel:              number of photons simulated per bixel
+%   numOfParallelMCSimulations: number of simultaneously performed simulations (optional) 
+%
+% output
+%   dij:        matRad dij struct
+%
+% References
+%   
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% set output level. 0 = no vmc specific output. 1 = print to matlab cmd. 
+% 2 = open in terminal(s)
+verbose = 0;
+
+% meta information for dij
+dij.numOfBeams         = pln.numOfBeams;
+dij.numOfVoxels        = pln.numOfVoxels;
+dij.resolution         = ct.resolution;
+dij.numOfRaysPerBeam   = [stf(:).numOfRays];
+dij.totalNumOfRays     = sum(dij.numOfRaysPerBeam);
+dij.totalNumOfBixels   = sum([stf(:).totalNumOfBixels]);
+dij.dimensions         = pln.voxelDimensions;
+dij.numOfScenarios     = 1;
+
+% set environment variables for vmc++
+if exist('vmc++','dir') ~= 7
+    error(['Could not locate vmc++ directory. ' ...
+           'Please download vmc++ from http://www.cerr.info/download.php and put it in your matRad root folder']);
```

Comment:
'Could not locate vmc++ directory. ' ...
'Please provide the binaries at matradroot/vmc++ (CERR hosts compatible binaries at http://www.cerr.info/download.php).'